src/dashboard/overview/ServiceDetails.tsx

```diff
@@ -1,49 +1,146 @@
-import { createStyles, makeStyles, Theme } from "@material-ui/core";
+import { makeStyles, Theme } from "@material-ui/core";
 import React, { ReactElement } from "react";
-import Dialog from "../../common/components/layout/dialog/Dialog";
+import { Subject } from "rxjs";
 import { drawerWidth } from "../../common/components/navigation/Menu";
 import { Status } from "../../models/Status";
 import ServiceDetailsContent from "./ServiceDetailsContent";
+import { formatDateTimeForFilename } from "../../common/utils/dateUtil";
+import api from "../../api";
+import DialogTitle from "@material-ui/core/DialogTitle";
+import Dialog from "@material-ui/core/Dialog";
```

Comment:
There's also a Dialog component in this project. Would really make it lot easier to maintain to use this.

Reply:
I found it very annoying if I want to design the dialog title part then I will have to give like 3 classes as props, is it necessary to have internal Dialog component?

Reply:
Wouldn't it be an option to give the title as an input instead?